### PR TITLE
merge .NET Standard and .NET Full builds 

### LINKED ISSUE #908: merge .NET Standard and .NET Full builds
We currently have

* build.cmd
* rakefile.rb
* FakeItEasy.sln
* FakeItEasy.nuspec

and *.netstd.* versions of the same.

AppVeyor builds both `build*.cmd`s.
The `FakeItEasy.netstd.nuspec` packages everything that's in `FakeItEasy.nuspec`.

We should 

- move the netstd versions over top of the others
- update AppVeyor to run the remaining build.cmd, and
- restore the proper version generation in the (soon to be) `rakefile.rb`

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #908.

I made many more commits than necessary, such as removing files, then moving their replacements on top instead of just renaming, to help git figure out what's really changing at each stage. I hope it helped the diffs.

I think it would be appropriate to squash all the commits later, but am happy to do whatever.

### CODE REVIEW

**Comment by thomaslevesque** [comment]
Looks good to me 👍 

> I made many more commits than necessary, such as removing files, then moving their replacements on top instead of just renaming, to help git figure out what's really changing at each stage. I hope it helped the diffs.

It did, thanks!

**Comment by thomaslevesque** [comment]
> I think it would be appropriate to squash all the commits later, but am happy to do whatever.

Yes, it should probably be squashed. But let's let @adamralph have a look first, if he wants to.

File: build.cmd [hunk 1]
```diff
@@ -1,3 +1,3 @@
 @echo off
 
-call bundle exec rake %*
+call bundle exec rake -f rakefile.rb %*
```
Comment (adamralph) [inline build.cmd]: This is redundant, but I guess it doesn't hurt to be explicit.

**Comment by adamralph** [comment]
Thanks @blairconrad !

**Comment by blairconrad** [comment]
Thanks for the reviews, guys. You're too quick for the squashing all over today, @adamralph. :wink:

**Comment by adamralph** [comment]
Oops! I did it again...

**Comment by blairconrad** [comment]
It's kind of nice, seeing that you're human!

**Pull request merged**